pkg/reconciler/route/route.go

```diff
@@ -118,7 +118,11 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, r *v1.Route) pkgreconcil
 	traffic, err := c.configureTraffic(ctx, r)
 	if traffic == nil || err != nil {
 		if err != nil {
-			r.Status.MarkUnknownTrafficError(err.Error())
+			if errors.As(err, &domains.DomainNameError{}) {
+				r.Status.MarkRevisionTargetTrafficError("ErrorConfig", err.Error())
```

Comment:
Do we have constants for `ErrorConfig` somewhere? Or why do we even need the reason as there is only one value here?